Optional combine_pdf + support yarn pnp (#266)
Studiosity/grover: README.md

end
```

> #### Yarn pnp strategy

Reviewer: ```suggestion
#### Yarn PnP strategy
```

> If you ar using yarn pnp strategy, you can override how to run js runtime for grover:

Reviewer: ```suggestion
If you are using the Yarn PnP strategy, you can override the run JS runtime for grover:
```

### Using middleware

> To use this functionality you need add [combine_pdf](https://rubygems.org/gems/combine_pdf) gem in your app.

Reviewer: ```suggestion
Note, to use this functionality you need to add the [combine_pdf](https://rubygems.org/gems/combine_pdf) gem to your app.
```

Also, I think this paragraph should be moved down one (just below the "You can specify relative paths to the cover page contents using..." one).
